packages/flutter_markdown/test/markdown_test.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
```

Comment:
How about calling this file (and the golden snapshot) `combination_test.dart` or `inline_test.dart`